glad/generator/rust/templates/impl.rs

```diff
@@ -1,5 +1,5 @@
 {% import 'template_utils.rs' as template_utils with context %}
-
+{{ '#![allow(non_snake_case)]' if options.mx }}
```

Comment:
Just allows allow, makes the template more readable and doesn't matter in the generated code

Reply:
Makes sense :) Should be fixed in https://github.com/Dav1dde/glad/pull/256/commits/3c1d7934900137cecf580409a64b890e30e97a1c.